pandas/tests/io/test_parquet.py

```diff
@@ -512,6 +512,19 @@ def test_basic_subset_columns(self, pa, df_full):
             read_kwargs={"columns": ["string", "int"]},
         )
 
+    def test_to_bytes_without_path_or_buf_provided(self, pa, df_full):
+        # GH 37105
+
+        buf = df_full.to_parquet(engine=pa)
+        assert isinstance(buf, bytes)
+
+        with tm.ensure_clean() as path:
+            with open(path, "wb") as f:
+                f.write(buf)
```

Comment:
I would maybe rather test that you can directly read the bytes again (by wrapping it in a BytesIO?) instead of writing the bytes to a file.

Reply:
Done